website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -209,24 +209,6 @@ aws ec2 create-key-pair --key-name <PREFIX> --query 'KeyMaterial' --output text
 Make sure your AWS is configured to use the cumulus sandbox account by using that account's AWS access keys before creating the EC2 key pair.
 :::
 
-After hitting the play button on `Deploy Dev Cumulus and ORCA Stack`, but before hitting `Run` in the popup, replace the following variables with yours.
-
-- CUMULUS_AWS_ACCESS_KEY_ID
-- CUMULUS_AWS_SECRET_ACCESS_KEY
-- PREFIX
-- AWS_ACCOUNT_ID (for cumulus sandbox account)
-- AWS_SUBNET_ID1
-- AWS_SUBNET_ID2
-- VPC_ID
-- EARTHDATA_CLIENT_ID
-- EARTHDATA_CLIENT_PASSWORD
-- DB_ADMIN_PASSWORD
-- DB_USER_PASSWORD
-- DB_HOST_ENDPOINT
-- RDS_SECURITY_GROUP
-- RDS_USER_ACCESS_SECRET_ARN
-- CUMULUS_ORCA_DEPLOY_TEMPLATE_VERSION (optional)
-
-The RDS variables `RDS_SECURITY_GROUP`, `RDS_USER_ACCESS_SECRET_ARN` and `DB_HOST_ENDPOINT` can be found from output logs of the previous `Deploy Dev RDS Stack` stage as `security_group_id`, `admin_db_login_secret_arn`, and `rds_endpoint` respectively. Note that a new earthdata application will need to be first created if using a new prefix for new deployment which will give the values for `EARTHDATA_CLIENT_ID` and `EARTHDATA_CLIENT_PASSWORD`. `CUMULUS_ORCA_DEPLOY_TEMPLATE_VERSION` is the branch you want to check out in the [deployment repo](https://git.earthdata.nasa.gov/projects/ORCA/repos/cumulus-orca-deploy-template/browse) such as `v11.1.1-v4.0.1`.
+A new earthdata application will need to be first created if using a new prefix for new deployment which will give the values for `EARTHDATA_CLIENT_ID` and `EARTHDATA_CLIENT_PASSWORD`. `CUMULUS_ORCA_DEPLOY_TEMPLATE_VERSION` is the branch you want to check out in the [deployment repo](https://git.earthdata.nasa.gov/projects/ORCA/repos/cumulus-orca-deploy-template/browse) such as `v11.1.1-v4.0.1`.

```

Comment:
We should only need to create the earthdata clientid and password once and then they can be used over and over again. The most you may have to do as an end user is update the redirect url for the application once this is setup. No need to create a new one each time you change a prefix. 
![Screen Shot 2022-08-10 at 3 01 58 PM](https://user-images.githubusercontent.com/2954734/184009471-ce5450c0-d607-497c-8777-1b5da5b91e4e.png)

In reality this is really only needed if you are calling the Cumulus API or doing things from the dashboard. In a majority of our integration tests so far, we are calling things directly from within the account so this setup may not be needed.

Definitely worth some discussion though as a team.

Reply:
I can update the wording to say users don't have to create a new application if they already have one.

Reply:
will keep the earthdata client ID